### PR TITLE
Only output blocklist warning if 410, provide helpful error message otherwise

### PR DESCRIPTION
**Pull request by nessup**

### CODE REVIEW

**Review by joaquim-verges — APPROVED** [review]

**Comment by jnsdls** [comment]
missing changeset & tests are failing

**Comment by joaquim-verges** [comment]
can just port this in the mega PR

File: packages/storage/src/core/downloaders/storage-downloader.ts [hunk 1]
```diff
@@ -54,9 +54,14 @@ export class StorageDownloader implements IStorageDownloader {
       }
 
       console.warn(
-        `Request to ${resolvedUri} failed with status ${resOrErr.status} - ${resOrErr.statusText}`,
+        `Request to ${resolvedUri} failed with status ${resOrErr.status} - ${resOrErr.statusText}}`,
```
Comment (iketw) [inline packages/storage/src/core/downloaders/storage-downloader.ts:57]: Is there an extra closing bracket here?

**Comment by joaquim-verges** [comment]
done in main already

**Pull request closed without merging**